docusaurus.config.js

```diff
@@ -47,8 +47,8 @@ module.exports = {
           label: 'Reference',
         },
         {
-          to: '/dashboard',
-          label: 'Go to Dashboard',
+          to: '/',
+          label: 'Go to App',
```

Comment:
What App? I think this label is confusing, notably because you get different result if you are logged in / out.

How about just putting `superface.ai` there?

```suggestion
          label: 'superface.ai',
```

Reply:
Suggestion is confusing as well, because you are already on superface.ai. 🤷

Reply:
So how about pointing to catalog? I didn't want to use it, because I want to avoid using catalog in header, but ow it may be the best.

Reply:
Sounds good!